paper/src/main/resources/config.yml

```diff
@@ -2,6 +2,14 @@
 # system messages. This is what makes chat not reportable.
 rewrite-chat: true
 
+# Whether FreedomChat should claim to clients that secure chat is enforced.
+# With this set to true, the "Chat messages can't be verified" toast will not
+# be shown. This is, in default configurations, unrelated to allowing clients
+# not signing their messages join. In modern versions, clients also
+# require a valid access token to be present for the popup to be hidden.
```

Comment:
nitpick: keep the "toast" wording here instead of popup to be more clear that it's talking about the same thing

Reply:
Changed